django_admin_row_actions/admin.py

```diff
@@ -30,14 +17,14 @@ class AdminRowActionsMixin(object):
 
     @property
     def media(self):
-        return super(AdminRowActionsMixin, self).media + forms.Media(
+        return super().media + forms.Media(
             css={'all': ["css/jquery.dropdown.min.css"]},
             js=["js/jquery.dropdown.min.js"],
         )
 
     def get_list_display(self, request):
         self._request = request
```

Comment:
see # _request

Reply:
Not invalid, but outside the scope of this PR.

Also not a huge deal IMO since the only place it is used is in `_row_actions` (which must be its own method due to how Django's admin works), and the only place `_row_actions` is ever referenced is in the next three lines.

If you'd like to submit an issue to track it or a PR fixing it, please feel free. But it's probably not something I'm going to personally fix.